string rover_name
bool notify_others
---